raiden/network/proxies/netting_channel.py

```diff
@@ -448,9 +445,9 @@ def settle(self):
 
     def events_filter(
             self,
-            topics: Optional[List],
-            from_block: Optional[int] = None,
-            to_block: Optional[int] = None
+            topics: List[str] = None,
+            from_block: int = None,
+            to_block: int = None
```

Comment:
shouldn't these also be `Union[str, int]`? If so, could you give a name for this type on the typing module?

Reply:
Done